app/uk.gov.hmrc.helptosaveproxy/connectors/NSIConnector.scala

```diff
@@ -145,6 +148,17 @@ class NSIConnectorImpl @Inject() (conf: Configuration, metrics: Metrics, pagerDu
       }
   }
 
+  override def getAccountByNino(nino: NINO, version: String, systemId: String, correlationId: UUID)(implicit hc: HeaderCarrier, ex: ExecutionContext): Result[HttpResponse] = {
+
+    val url = s"$nsiGetAccountByNinoUrl?nino=$nino&version=$version&systemId=$systemId&correlationId=$correlationId"
```

Comment:
Query parameter values should be escaped or validated to prevent problems when special characters are included in the String parameters.

If the [controller](https://github.com/hmrc/help-to-save-proxy/pull/15/files#r173225278) gets the `request.rawQueryString` and passes it in to this method then no escaping will be required (because the parameters will not have been un-escaped).